src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/TestUriHelper.cs

```diff
@@ -36,6 +36,12 @@ public static int FindFreePort(int initialPort)
         {
```

Comment:
I think `FindFreePort(int initialPort)` should be removed, leaving only `FindFreePort()`.  There's no reason for a caller to specify an initial port.